src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -631,5 +658,33 @@ public ApplicationViewModel.AffordabilityViewModel GetAffordabilityViewModel(int
                 YearDescription = _dateTimeService.UtcNow.ToTaxYearDescription()
             };
         }
+
+        private List<string> GetRejectedApplicationsDasNames(List<string> rejectedApplications, GetRejectApplicationsResponse applicationsWithDasNames)
+        {
+            var getDasAccountNames = new List<string>();
+
+            foreach (var appplicationId in rejectedApplications)
+            {
+                foreach (var application in applicationsWithDasNames.Applications)
+                {
+                    if (application.Id == (int)_encodingService.Decode(appplicationId, EncodingType.PledgeApplicationId))
+                    {
+                        getDasAccountNames.Add(application.DasAccountName);
+                    }
+                }
+            }
+            return getDasAccountNames;
+        }
+
+        private IEnumerable<int> GetDecodedApplicationsIds(List<string> applicationsIds)
```

Comment:
This also looks like a single-use method, which doesn't seem worthwhile. Perhaps an extension method or similar might be cleaner?

Reply:
Sure, I have removed this private method now.